Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -163,23 +160,27 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
      traversed section of a route, override the `traversedRouteColor` property
      for the `NavigationMapView.appearance()`.
      */
-    public var routeLineTracksTraversal: Bool = false {
-        didSet {
-            mapViewController?.routeLineTracksTraversal = routeLineTracksTraversal
+    public var routeLineTracksTraversal: Bool {
+        get {
+            navigationMapView?.routeLineTracksTraversal ?? false
+        }
+        set {
+            navigationMapView?.routeLineTracksTraversal = newValue
         }
     }
 
     /**
      Controls whether or not the FeedbackViewController shows a second level of detail for feedback items.
      */
-    public var detailedFeedbackEnabled: Bool = false {
-        didSet {
-            mapViewController?.detailedFeedbackEnabled = detailedFeedbackEnabled
+    public var detailedFeedbackEnabled: Bool {
```

Comment:
Since the default value is no longer part of the public type declaration, mention the default value in the documentation comment (also for `annotatesSpokenInstructions`).